sources/oracle/README.md

```diff
@@ -23,7 +23,7 @@ retrieve table schema and data. Set the `-source=oracle` and corresponding
 source profile connection parameters `host`, `port`, `user`, `dbName` and
 `password`.
 
-For example to perform schema conversion, run
+For example to perform schema migration, run
```

Comment:
Nit: Add a comma after For example -
For example,  to perform...

(Applicable to other places as well in the PR)